### ISSUE TITLE
Autocomplete and shortcut keys not working on Ubuntu 18.04 Firefox 65

### ISSUE THREAD

**Issue by 2lone2bone** [jupyterlab/jupyterlab#5963]
I have this strange problem that autocomplete (python) nor shortcut keys are not working in Ubuntu (18.04) and Firefox (65.0). I have all the libraries for autocomplete installed. I am running jupyter within a virtualenv running python 3.6. When using chrome the shortcut keys do work. I have webrtc disabled and fingerprint resist on in Firefox which should not play a role here I guess.

**Comment by jasongrout** [comment]
Do you have any extensions loaded in Firefox? Can you try disabling any extensions and running it in Private mode?

**Comment by 2lone2bone** [comment]
I disabled all the extensions and ran it in private mode but still for example ctrl+' is not working for single line comment. This works flawlessly on chromium though.

**Comment by WesleyTheGeolien** [comment]
I have tried JupyterLab in Ubuntu 18.04 + Firefox 65.0

For `Ctrl+f` if I am editing a cell then I get cell search and if I am in "explore" (not quite sure what the correct term is but when you hit `Esc` then `Ctrl+f` opens the default firefox find in page command.

As for `ctrl+'` On my setup, this doesn't comment a line in Firefox or Chrome (I don't have Chromium) @jasongrout is this a default command? @2lone2bone could you check your shortcuts to confirm this should be comment a line? Personally `Ctrl+/` comments one line.

**Comment by 2lone2bone** [comment]
@WesleyTheGeolien the problem is that there is no settings that even remotely would be related to commenting. It just works with Chromium and not with Firefox. I have a Finnish (should have mentioned in the original post sorry) layout so that might be the reason for the different combo (`ctrl + /` does not work for me since `/` is behind shift anyways) and the location is almost the same for `/` in US layout and for  `'` in Finnish layout.
The odd thing with Firefox is that the most (not all) of the shortcuts in the "Commands" tab are grayed out and uneditable whereas in Chromium most are white and editable. If you have a specific path to that setting could you please advise me and I could try it out.

**Comment by newfylox** [comment]
Same problem but not the same browser. I don't know if it will help you guys but disabling "Google Play Music" extension on Chrome version 75 did the trick!

**Comment by LoudeNOUGH** [comment]
Suddenly all the shortcuts, except Shift+Enter are disabled. You cannot enter a cell with Enter or escape it with Esc. In cell edit mode all the letter-keys are functioning, but even Shift+Enter works only if you escape the cell using the help of cursor.

It happened before, and restart of jupyter helped, but not this time.

The same happens when using Safari and Chrome. JupyterLab is the latest (1.0.9) version

**Comment by mullanptr** [comment]
I have the same issue - Most Shortcuts (like 'Esc' to leave insert mode, or 'Shift'+'Enter' to run cell) are not working anymore.

Yesterday, I upgraded my firefox to FF70, on my Ubuntu 16.04.
Jupyter lab was working fine, just after this update.
But not today, anymore. I made some restarts of my machine inbetween, if that is of any help...

**Comment by RobertoMaurizzi** [comment]
@LoudeNOUGH  and @mullanptr mullanptr if you have OSX and use multiple monitors/desktop, it might be that you commanded a "navigate away" or "reload" of the page and you have a "Reload site?" dialog open somewhere but absolutely not visible on the screen: the modal dialog comes up in a different virtual desktop on a different virtual monitor
I had that problem many times, even if right now with Chrome 78 and Catalina it's either fixed (the dialog comes up in the middle of Chrome's active window) or much more difficult to trigger.

**Comment by LoudeNOUGH** [comment]
@RobertoMaurizzi No, I use neither multiple monitors no multiple desktops. The issue happened during the most simple use case.

**Comment by jasongrout** [comment]
Looking through the history of comments here, my guess is that there have been a number of different issues lumped together here, some with issues in the environment, and possibly some with jlab as well. My point is that I'm not sure that we can assume that relevant information for one person is also relevant for solving another person's issue.

@RobertoMaurizzi - can you go through the issue diagnosis guide at https://jupyterlab.readthedocs.io/en/stable/getting_started/issue.html and (a) see if it helps diagnose the issue and (b) tell us the results of each step (like what js errors you see, etc.)?

**Comment by Serhiy-Shekhovtsov** [comment]
This is super weird. I had the same issue, at some point Up, Down and Esc keys stopped working for navigation. When I am within the cell Up and Down are working, but can't exist using Esc.
What solved the issue is stopping the JupyterLab and than removing all `.ipynb_checkpoints` folders in my project. Then started the JupyterLab and the issue was gone.
